docs/en/engines/database-engines/materialized-postgresql.md

```diff
@@ -25,25 +25,54 @@ ENGINE = MaterializedPostgreSQL('host:port', ['database' | database], 'user', 'p
 
 ## Settings {#settings}
 
--   [materialized_postgresql_max_block_size](../../operations/settings/settings.md#materialized-postgresql-max-block-size)
-
 -   [materialized_postgresql_tables_list](../../operations/settings/settings.md#materialized-postgresql-tables-list)
 
+-   [materialized_postgresql_schema](../../operations/settings/settings.md#materialized-postgresql-schema)
+
 -   [materialized_postgresql_allow_automatic_update](../../operations/settings/settings.md#materialized-postgresql-allow-automatic-update)
 
+-   [materialized_postgresql_max_block_size](../../operations/settings/settings.md#materialized-postgresql-max-block-size)
+
 -   [materialized_postgresql_replication_slot](../../operations/settings/settings.md#materialized-postgresql-replication-slot)
 
 -   [materialized_postgresql_snapshot](../../operations/settings/settings.md#materialized-postgresql-snapshot)
 
 ``` sql
 CREATE DATABASE database1
 ENGINE = MaterializedPostgreSQL('postgres1:5432', 'postgres_database', 'postgres_user', 'postgres_password')
-SETTINGS materialized_postgresql_max_block_size = 65536,
-         materialized_postgresql_tables_list = 'table1,table2,table3';
+SETTINGS materialized_postgresql_tables_list = 'table1,table2,table3';
 
 SELECT * FROM database1.table1;
 ```
 
+PostgreSQL [schema](https://www.postgresql.org/docs/9.1/ddl-schemas.html) can be used in two ways.
+
+1. One schema for one `MaterializedPostgreSQL` database engine. Requires to use setting `materialized_postgresql_schema`.
+Tables are accessed via table name only:
+
+``` sql
+CREATE DATABASE postgres_database
+ENGINE = MaterializedPostgreSQL('postgres1:5432', 'postgres_database', 'postgres_user', 'postgres_password')
+SETTINGS materialized_postgresql_schema = 'postgres_schema';
```

Comment:
Thanks! We are really expecting this feature. I have a suggestion regarding Materializing multiple schemas in a single database.

`materialized_postgresql_schema` restricts only one schema and one had to use `materialized_postgresql_tables_list` if multiple schemas has to be materialized. It would be really tedious & might cause issues for hundreds of tables in each schema (out use case) if it has to be represented in `materialized_postgresql_tables_list`.

Can this be simplified by introducing `materialized_postgresql_schema_list` option?

Reply:
> Can this be simplified by introducing materialized_postgresql_schema_list option?

It can be possible in case you want to replicate all tables from all those schemas. Is it what you mean?

Reply:
Ok, I'll add this also.

Reply:
is it possible to replicate many tables in postgres schemas into one table in clickhouse

Reply:
Not possible at the moment, but you can create a github issue feature-request for it.